Makefile

```
@@ -1,44 +1,79 @@
 RSYNC := rsync -ltvrP --delete --exclude private/ --perms --chmod=Fugo+r,Fug+w,Dugo-s,Dugo+rx,Dug+w
 RSYNC_BASE := rsync://feed.community.greenbone.net:
 VERSION := 22.04
-NASL_TARGET := /var/lib/openvas/plugins/
-NOTUS_TARGET := /var/lib/notus/
-SC_TARGET := /var/lib/gvm/data-objects/gvmd/22.04/scan-configs/
-
-check-feed-dirs:
-	@ [ -d ${NASL_TARGET} ] || (printf "\e[31m${NASL_TARGET} missing\e[0m\n" && false)
-	@ [ -d ${NOTUS_TARGET} ] || (printf "\e[31m${NOTUS_TARGET} missing\e[0m\n" && false)
-	@ [ -d ${SC_TARGET} ] || (printf "\e[31m${SC_TARGET} missing\e[0m\n" && false)
-	@ test -w ${NASL_TARGET} || (printf "\e[31m${USER} cannot write into ${NASL_TARGET}\e[0m\n" && false) 
-	@ test -w ${NOTUS_TARGET} || (printf "\e[31m${USER} cannot write into ${NOTUS_TARGET}\e[0m\n" && false) 
-	@ test -w ${SC_TARGET} || (printf "\e[31m${USER} cannot write into ${SC_TARGET}\e[0m\n" && false) 
+
+ifndef INSTALL_PREFIX
+	INSTALL_PREFIX := /var
+endif
+
+NASL_TARGET_DEFAULT := ${INSTALL_PREFIX}/lib/openvas/plugins
+nasl_target := ${NASL_TARGET_DEFAULT}
+
+NOTUS_TARGET_DEFAULT := ${INSTALL_PREFIX}/lib/notus
+notus_target := ${NOTUS_TARGET_DEFAULT}
+
+SC_TARGET_DEFAULT := ${INSTALL_PREFIX}/lib/gvm/data-objects/gvmd/22.04/scan-configs
+sc_target := ${SC_TARGET_DEFAULT}
```

```suggestion
ifndef INSTALL_PREFIX
	INSTALL_PREFIX := 
endif

NASL_TARGET_DEFAULT := ${INSTALL_PREFIX}/var/lib/openvas/plugins
nasl_target := ${NASL_TARGET_DEFAULT}

NOTUS_TARGET_DEFAULT := ${INSTALL_PREFIX}/var/lib/notus
notus_target := ${NOTUS_TARGET_DEFAULT}

SC_TARGET_DEFAULT := ${INSTALL_PREFIX}/var/lib/gvm/data-objects/gvmd/22.04/scan-configs
sc_target := ${SC_TARGET_DEFAULT}
```
The `INSTALL_PREFIX` does not contain `/var`